### PR TITLE
Update Documentation for Windows installation

### PR DESCRIPTION
**Pull request by jonathanzhang53**
Updated the Installation docs

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -256,175 +294,111 @@ AllowOverride All
...
-**Q. How can I install Apache, MySQL, and PHP on Linux?**
-
-Follow the instructions [here](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-on-ubuntu-16-04)
-
+Follow the instructions [here](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-on-ubuntu-16-04).
 
 **Q. I'm getting `table doesn't exist` error!**
 
-This is because MySQL versions 5.7 and above have `strict mode` on default which needs to be disabled. This can be done by editing the MySQL configuration file and appending `sql_mode = ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION` at the end (if it is already present, modify it).
+This is because mySQL versions 5.7 and above have `strict mode` on default which needs to be disabled. This can be done by editing the mySQL configuration file and appending `sql_mode = ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION` at the end (if it is already present, modify it).
```
Comment (robbyoconnor) [inline INSTALL.md]: The previous was correct, please revert.

File: INSTALL.md [hunk 2]
```diff
@@ -256,175 +294,111 @@ AllowOverride All
...
 **Q. How do I restart the apache service?**
 
-Restart apache service by using `sudo apache2ctl restart` on a terminal for Linux.
-For Windows, restart apache service by using the XAMPP control interface, located in system tray (if running) or from `xampp-control.exe` in `C:\xampp`. You can restart apache by navigating to `C:\xampp\apache\bin` using CMD and executing `httpd -k restart`.
-
+[Linux] Restart apache service by using `sudo apache2ctl restart` on a terminal.
+[Windows] Restart apache service by using the XAMPP control interface, located in system tray (if running) or from `xampp-control.exe` in `C:\xampp`. You can restart apache by navigating to `C:\xampp\apache\bin` using CMD and executing `httpd -k restart`.
 
-**Q. I am getting a Internal Server Error when trying to run portal dashboard!**
+**Q. I'm getting a Internal Server Error when trying to run portal dashboard!**
```
Comment (robbyoconnor) [inline INSTALL.md]: Please revert.

File: INSTALL.md [hunk 3]
```diff
@@ -1,73 +1,144 @@
...
-**[Overview of Directories](#overview-of-directories)**
-
 **[Unpacking](#unpacking)**
 
-**[Setup](#setup)**
+**[Windows Setup](#windows-setup)**
+
+**[Non XAMPP Setup](#non-xampp-setup)**
+
+**[Disabling mySQL Strict Mode](#how-do-i-disable-mysql-strict-mode-?)**
```
Comment (robbyoconnor) [inline INSTALL.md]: MySQL

File: INSTALL.md [hunk 4]
```diff
@@ -1,73 +1,144 @@
...
 
 **[Upgrading](#upgrading)**
 
 **[FAQ](#faq)**
 
-**[Windows Setup](#windows-setup)**
+##  Unpacking
 
-##  Overview of Directories
+Download the source code directly from the repository located at [lh-ehr](https://github.com/LibreHealthIO/librehealthehr) using git bash.
```
Comment (robbyoconnor) [inline INSTALL.md]: Wrong repo name.

File: INSTALL.md [hunk 5]
```diff
@@ -1,73 +1,144 @@
...
 
-Alternatively you can download the source code directly from the repository located at [librehealthehr](https://github.com/LibreHealthIO/librehealthehr) using
+##  Windows Setup
 
-```
-git clone https://github.com/LibreHealthIO/LibreEHR librehealthehr
-```
+#### Requirements
+
+To run LibreEHR on Windows, [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is needed. Make sure it is downloaded to 'C:\' directory to give XAMPP full permissions.
```
Comment (robbyoconnor) [inline INSTALL.md]: LibreHealth EHR please

File: INSTALL.md [hunk 6]
```diff
@@ -1,73 +1,144 @@
...
+ display_errors = Off
+ upload_tmp_dir is set to a correct default value that will work on your system
+ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE
+ ```
 
-##  Setup
+4. Save the file and close it.
 
-To run LibreHealthEHR, MariaDB (prefered) or MySQL, and Apache or another PHP-capable webserver must be configured.
+5. Make sure that strict mode is disabled in mySQL. Instructions [here](#how-do-i-disable-mysql-strict-mode-?).
```
Comment (robbyoconnor) [inline INSTALL.md]: MySQL

File: INSTALL.md [hunk 7]
```diff
@@ -1,73 +1,144 @@
...
+To run LibreHealthEHR, MariaDB (prefered) or mySQL, and Apache or another PHP-capable webserver must be configured.
+
+If you don't already have it, download and install [Apache](https://www.apachelounge.com/download/), [MariaDB](https://downloads.mariadb.org/) (prefered) or [mySQL](https://www.mysql.com/downloads/), and [PHP.](http://php.net/downloads.php)
 
 **Note:**
 
 1. PHP versions 7.1+ are not supported.
 
-2. MySQL versions 5.7+ have strict mode enabled by default and must be disabled. Instructions on how to disable it are given in the FAQs section.
+2. mySQL versions 5.7+ have strict mode enabled by default and must be disabled. Instructions [here](#how-do-i-disable-mysql-strict-mode-?).
```
Comment (robbyoconnor) [inline INSTALL.md]: MySQL

File: INSTALL.md [hunk 8]
```diff
@@ -1,73 +1,144 @@
...
+#### Requirements
+
+To run LibreHealthEHR on Windows, [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is needed. Make sure it is downloaded to 'C:\' directory to give XAMPP full permissions.
+
+**Note:**
+
+1. PHP [7.0](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/_) works and so does [5.6](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/) but [5.6.30](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is recommended
+
+2. PHP 7.1x is not currently supported
+
```
Comment (robbyoconnor) [inline INSTALL.md]: You are recommending [unsupported versions of PHP](http://php.net/eol.php). At this point, PHP 7.1+ has to work.

File: INSTALL.md [hunk 9]
```diff
@@ -178,66 +240,42 @@ display_errors = Off
 upload_tmp_dir is set to a correct default value that will work on your system
 error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE
 ```
-Make sure that settings in MYSQL /etc/mysql/my.cnf file include:
+- Make sure that settings in MySQL/etc/mysql/my.cnf file include:
```
Comment (robbyoconnor) [inline INSTALL.md]: Why did you remove the space here?

**Comment by robbyoconnor** [comment]
@konikodes -- you wanna have a peek?

File: INSTALL.md (hunk 9 above)
Comment (jonathanzhang53) [reply INSTALL.md]: It wasn't consistent with the rest of the document. This is what it looks like now.
image

Comment (robbyoconnor) [reply INSTALL.md]: You can't do that. That is a file path.

Comment (jonathanzhang53) [reply INSTALL.md]: Oh you are talking about that space. The file path shouldn't have a space in it anyway though right?

File: INSTALL.md [hunk 10]
```diff
@@ -256,175 +288,112 @@ AllowOverride All
...
 This is because MySQL versions 5.7 and above have `strict mode` on default which needs to be disabled. This can be done by editing the MySQL configuration file and appending `sql_mode = ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION` at the end (if it is already present, modify it).
 
 In Linux, this file is located in `/etc/mysql/mysql.conf.d/mysqld.cnf`. In Windows, it is usually located in `C:\ProgramData\MySQL\MySQL Server 5.7\my.ini`
 
-
 **Q. How do I restart the apache service?**
 
-Restart apache service by using `sudo apache2ctl restart` on a terminal for Linux.
-For Windows, restart apache service by using the XAMPP control interface, located in system tray (if running) or from `xampp-control.exe` in `C:\xampp`. You can restart apache by navigating to `C:\xampp\apache\bin` using CMD and executing `httpd -k restart`.
+[Linux] Restart apache service by using `sudo apache2ctl restart` on a terminal.
```
Comment (robbyoconnor) [inline INSTALL.md]: Not how you make a hyperlink in markdown.

File: INSTALL.md [hunk 11]
```diff
@@ -1,73 +1,138 @@
...
 
 **Note:**
 
-1. PHP versions 7.1+ are not supported.
+1. PHP versions must be 7.1 or 7.2.
 
-2. MySQL versions 5.7+ have strict mode enabled by default and must be disabled. Instructions on how to disable it are given in the FAQs section.
+2. MySQL versions 5.7+ have strict mode enabled by default and must be disabled. Instructions [here](#how-do-i-disable-mysql-strict-mode-?).
 
 3. LibreHealthEHR requires a number of webserver and PHP features which may not be enabled on your system.  These include:
```
Comment (robbyoconnor) [inline INSTALL.md]: There's a space. between LibreHealth and EHR.

File: INSTALL.md [hunk 12]
```diff
@@ -1,73 +1,138 @@
...
   * PHP Index support (ensure that index.php is in your Index path in httpd.conf)
   * Session variables
   * PHP libcurl support (optional for operation, mandatory for billing)
 
-4. If installing on Linux, make sure these dependencies are met:
+4. [Linux] Make sure these dependencies are met.
```
Comment (robbyoconnor) [inline INSTALL.md]: Break up the Linux vs Windows Install to separate files.

**Comment by jonathanzhang53** [comment]
For now I have just put to download the latest version of xampp (comes with latest php). I assume a migration from 5.6 to 7.1/7.2 is in order now that 5.6 is about to become unsupported. That line might have to change depending on what happens with the project.

**Comment by KoniKodes** [comment]
Does this file have a name?

> After unpacking lh-ehr, there is a `php` file with type 'Configuration Settings' located in `xampp\php\` directory. Open this file with your text editor.
 3. Change the following in the PHP file using `Ctrl+F`. Some of the variables shown appear multiple times in the file. Do not change any of the values above Line 170 (a blank line

**Comment by KoniKodes** [comment]
> For now I have just put to download the latest version of xampp (comes with latest php). I assume a migration from 5.6 to 7.1/7.2 is in order now that 5.6 is about to become unsupported. That line might have to change depending on what happens with the project.

I thought it could not be above 7.0

**Comment by jonathanzhang53** [comment]
> > For now I have just put to download the latest version of xampp (comes with latest php). I assume a migration from 5.6 to 7.1/7.2 is in order now that 5.6 is about to become unsupported. That line might have to change depending on what happens with the project.
> 
> I thought it could not be above 7.0

At this point, 5.6 and 7.0 are about to become unsupported versions of php. lh-ehr must migrate to 7.1 or 7.2 anyways, so it's up to the project team to to do that. I can't say to download unsupported versions of PHP in the documentation.

**Comment by jonathanzhang53** [comment]
> Does this file have a name?
> 
> > After unpacking lh-ehr, there is a `php` file with type 'Configuration Settings' located in `xampp\php\` directory. Open this file with your text editor.
> 
> 1. Change the following in the PHP file using `Ctrl+F`. Some of the variables shown appear multiple times in the file. Do not change any of the values above Line 170 (a blank line

The name of the file is `php`

**Comment by KoniKodes** [comment]
> > Does this file have a name?
> > > After unpacking lh-ehr, there is a `php` file with type 'Configuration Settings' located in `xampp\php\` directory. Open this file with your text editor.
> > 
> > 
> > 
> > 1. Change the following in the PHP file using `Ctrl+F`. Some of the variables shown appear multiple times in the file. Do not change any of the values above Line 170 (a blank line
> 
> The name of the file is `php`

I always thought "php" was an extension. Explains my confusion...

**Comment by jonathanzhang53** [comment]
> > > Does this file have a name?
> > > > After unpacking lh-ehr, there is a `php` file with type 'Configuration Settings' located in `xampp\php\` directory. Open this file with your text editor.
> > > 
> > > 
> > > 
> > > 1. Change the following in the PHP file using `Ctrl+F`. Some of the variables shown appear multiple times in the file. Do not change any of the values above Line 170 (a blank line
> > 
> > 
> > The name of the file is `php`
> 
> I always thought "php" was an extension. Explains my confusion...

php is a type of file but in this case it is the name of the file.

**Comment by KoniKodes** [comment]
That really explains my confusion...
